views/data_summary.erb

```diff
@@ -0,0 +1,355 @@
+          the system they are in does not use such mechanisms, we’ll also have
+          the <strong>area</strong> they represent, and the
+          <strong>group</strong> (perhaps a party or faction) they belong to.
+          There may also be start and end dates if we know they did not serve
+          the full term.
+        </p>
+        <a name="about-names"></a>
+        <h3>
+          About names
+        </h3>
+        <p>
+          Names are a key part of any data set based around people. In the
+          basic CVS, the <code>name</code> will be the common name that we’ve
+          got from the source, but we’re aware that names are not quite that
+          simple. We often have this name data:
+        </p>
+        <p>
+          <code>name</code>
+          <code>given_name</code>
+          <code>family_name</code>
+          <code>honorific_prefix</code>
+          <code>honorific_suffix</code>
+          <code>patronymic_name</code>
+          <code>sort_name</code>
+        </p>
+        <p>
+          However, in the CSV you’ll be getting these in whichever language was
+          available from the source of the data. Often this is the official
+          language of the country concerned, but that’s not always
+          straightforward because many countries have multiple official
+          languages. Furthermore, you can’t determine <em>which</em> language
+          it is from within the CSV — remember that CSV really is a coarse
+          version of the data — so instead you should use the JSON. In there
+          you’ll find that fields such as names are broken out by language code
+          (and often that will include transliteration in languages that are
+          not official languages of the country concerned, too).
+        </p>
+        <h3>
+          If you <em>only</em> want names
+        </h3>
+        <p>
+          For convenience, we isolate all the names of all the politicians
+          within each legislature and make them available in their own file.
+          (Like other files, look inside <code>countries.json</code> to find
+          its path and filename; in this case under <code>names</code>). This
+          file contains <code>name</code> and <code>id</code> fields. Note that
+          there may be more than one name for each politician (for example, we
+          might have the name of a Welsh politician rendered in English and
+          Chinese). If you need to consolidate them, use the <code>id</code> to
+          match them (or, as ever, use the JSON data instead).
+        </p>
+        <p>
+          In fact, we’ve even built an external service that consolidates all
+          the names into one CSV file (caution: that currently contains well
+          over 80,000 names and growing): see
+          <a href="https://github.com/everypolitician/everypolitician-names/">everypolitician-names</a>.
+        </p>
+        <h3>
+          IDs to other data sets, including WikiData
```

Comment:
WikiData shouldn't be camel-cased like this — it's just Wikidata (here and elsewhere)